ocf_datapipes/training/common.py

```diff
@@ -1265,12 +1265,23 @@ def create_t0_and_loc_datapipes(
     return location_pipe, t0_datapipe
 
 
-def potentially_coarsen(xr_data: xr.Dataset):
-    """Coarsen the data, if it is separated by 0.05 degrees each"""
+def potentially_coarsen(xr_data: xr.Dataset, coarsen_to_deg: float = 0.1):
+    """
+    Coarsen the data, if it is separated by 0.05 degrees each
+
+    Args:
+        xr_data: xarray dataset
+        coarsen_to_deg: Coarsen to this degree in lat and lon
+    """
     if "latitude" in xr_data.coords and "longitude" in xr_data.coords:
```

Comment:
Might be silly, but do we want to add a warning if latitude and longitude are not found and coarsening didn't happen? You'd want to know, especially bc later on you need to put adjusted crop shapes into the config